modules/ocf_mirrors/templates/systemd/sync.service.erb

```diff
@@ -0,0 +1,7 @@
+[Unit]
+Description=Sync service for <%= @title %>
+
+[Service]
+User=<%= @exec_user %>
+Type=simple
+ExecStart=<%= @exec_start %>
```

Comment:
You might want to set up email so that we continue to get emails when systemd timer-controlled jobs fail

Reply:
should be optional imo, and maybe not even needed for any of the mirrors cronjobs

a while back we decided that we just can't deal with cronjob spam from mirrors, they are just too inherently flaky (we're talking to ~a dozen servers that we don't control, they're gonna have downtime)

instead we took the approach of implementing healthchecks so we only get notified when it has failed consistently for a while, i think this is a much better approach for mirrors

(some other cronjobs make sense for emails though)

Reply:
I think for this we'd want to not have output as @chriskuehl said, since they can be pretty spammy if the upstream server is down. However, we'd definitely need health checks on all mirrors first, because otherwise we'd have no way of knowing if one of the non-ftpsync ones are way behind.